charts/kubeai/values.yaml

```diff
@@ -18,10 +18,9 @@ modelServers:
       # The "default" image should always be specified.
       # "default" is used when no imageName is specified or if a specific image is not found.
       default: "vllm/vllm-openai:v0.6.1.post2"
-      cpu: "substratusai/vllm:v0.5.5-cpu"
+      cpu: "ghcr.io/substratusai/vllm:v0.6.1-cpu"
```

Comment:
hmm these images tend to be much slower for me. I think in the past it was like 10x slower than using Docker Hub on GKE. GCP seems to cache Docker hub automatically but not ghcr.io